Add support openapi-generator mapping configuration properties:

altro3 opened the pull request:
schemaMapping
importMapping
nameMapping
typeMapping
enumNameMapping
modelNameMapping

Fixed #1421

sdelamo reviewed (commented):
can you include tests?

altro3 commented:
@sdelamo Well, what can I say, if I write a regular unit test, then it will not be testing our generator, but the openapi-generator library, because it performs all these functions. I'm not sure it's worth spending time on this.

I just added the ability to set these parameters through the configuration so that the user can set them through the plugin.

So, in order to fully test this mapping, you need to put all these properties in the gradle, but because of this, the generated code will completely change - all class names, mapping to other schemes, etc. If you add these tinctures to existing test suites, you will have to completely recycle them, because absolutely all names and the main the problem is, the generated code will simply be difficult to match with the source file.

I checked all this locally, everything worked out correctly, added only one property to the test - 
```groovy
nameMapping = [test: "changedTest"]
```
because it affects only one class. Prefixes and suffixes affect absolutely all generated code

sdelamo commented on openapi-generator/src/main/java/io/micronaut/openapi/generator/MicronautCodeGeneratorEntryPoint.java:
```diff
@@ -141,6 +142,46 @@ private void configureOptions() {
+            if (options.openapiNormalizer != null) {
+                javaCodeGen.addOpenapiNormalizer(options.openapiNormalizer);
+            }
+            if (options.apiNamePrefix != null && !options.apiNamePrefix.isBlank()) {
```
We have a utils class `StringUtils::isNotBlank`

altro3 replied:
I know, but this module does not depend on the libraries of the micronaut. It depends only on the openapi-generator
